### ISSUE TITLE
problem with Laravel 9

### ISSUE THREAD

**Issue by freemiumdev** [HalalSoft/laravel-dynamic-column#6]
Hi, i've a fresh installation of latest Laravel version,
when i try to install with composer i get this error:

` Problem 1
    - Root composer.json requires halalsoft/laravel-dynamic-column ^0.1.2 -> satisfiable by halalsoft/laravel-dynamic-column[v0.1.2].
    - halalsoft/laravel-dynamic-column v0.1.2 requires illuminate/database 6.*|7.*|8.* -> found illuminate/database[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.`

how can i solve?

thank you

**Comment by dyaskur** [comment]
I made an update for Laravel 9 support. Could you re-reinstall?

**Comment by freemiumdev** [comment]
Yes, and I confirm that it has now installed correctly. thank you
